cfy_manager/components/nginx/config/redirect-to-monitoring.cloudify

```diff
@@ -0,0 +1,23 @@
+location ~ ^/monitoring {
+    proxy_pass         http://cloudify-monitoring;
+    proxy_redirect     off;
+
+    proxy_set_header   Host              $host;
+    proxy_set_header   X-Real-IP         $remote_addr;
+    proxy_set_header   X-Server-Port     $server_port;
+    proxy_set_header   X-Forwarded-For   $proxy_add_x_forwarded_for;
+    proxy_set_header   X-Forwarded-Proto $scheme;
+
+    sub_filter_types text/html;
+    sub_filter_once off;
+    sub_filter '="/' '="/monitoring/';
+    sub_filter 'var PATH_PREFIX = "";' 'var PATH_PREFIX = "/monitoring";';
+
+    rewrite ^/monitoring/?$ /monitoring/graph; # `redirect` looses port in case of ssh tunnel
+    rewrite ^/monitoring/(.*)$ /$1 break;
```

Comment:
all of this seems weird, it's for path manipulation?

Reply:
Yes it is.  I could use `--web.external-url` Prometheus parameter instead.  Will look into this solution

Reply:
done